net/krb5/Makefile

```diff
@@ -25,6 +25,8 @@ PKG_INSTALL:=1
 PKG_CHECK_FORMAT_SECURITY:=0
 PKG_USE_MIPS16:=0
 
+PKG_BUILD_DEPENDS:=e2fsprogs/host
```

Comment:
This is redundant since the e2fsprogs makefile in master build-depends on its own host build already whiler the `+libss +libcomerr` depends further down will take care of pulling in e2fsprogs.

Reply:
Guess i still misunderstand something, can you explain how libss/libcomerr are pulling in e2fsprogs?